src/ngx_pagespeed.cc

```diff
@@ -1988,9 +1981,22 @@ void ps_send_to_pagespeed(ngx_http_request_t* r,
                           ngx_chain_t* in) {
   ngx_chain_t* cur;
   int last_buf = 0;
+
   for (cur = in; cur != NULL; cur = cur->next) {
     last_buf = cur->buf->last_buf;
 
+    if (cur->buf->flush && ctx->follow_flushes) {
```

Comment:
Does it ever make sense to flush on intermediate buffers in the chain, as opposed to the last buffer?